### PR TITLE
Prompt user to add build task if missing

### PR DESCRIPTION
**Pull request by DustinCampbell**
When the extension is activated and the workspace contains a project.json in the root path, we now check to see if there exists a `{workspaceRoot}/.vscode/tasks.json` file. If the file doesn't exist, or it exists but doesn't contain a build task, we prompt the user to add the build task.

cc @gregg-miskelly, @chuckries

### CODE REVIEW

File: src/tasks.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+ *--------------------------------------------------------------------------------------------*/
+
+'use strict';
+
+import * as fs from 'fs-extra-promise';
+import * as path from 'path';
+import * as vscode from 'vscode';
+import * as tasks from 'vscode-tasks';
+
+function promptToAddBuildTask() {
```
Comment (gregg-miskelly) [inline src/tasks.ts]: Define the type of the return?

File: src/tasks.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    fs.ensureDirAsync(vscodeFolder).then(() => {
+        fs.existsAsync(tasksJsonPath).then(exists => {
+            if (exists) {
+                fs.readFileAsync(tasksJsonPath).then(text => {
+                    const fileText = text.toString();
+                    let tasksConfig: tasks.TaskConfiguration = JSON.parse(fileText);
+                    let buildTask = tasksConfig.tasks.find(td => td.taskName === 'build');
+                    if (!buildTask) {
+                        promptToAddBuildTask().then(shouldAdd => {
+                            buildTask = createBuildTaskDescription();
```
Comment (gregg-miskelly) [inline src/tasks.ts]: Does this code need to be in an 'if (shouldAdd) {' ...?

**Comment by gregg-miskelly** [comment]
Signing off with comments.

File: src/tasks.ts (hunk 1 above)
Comment (DustinCampbell) [inline src/tasks.ts]: Thanks!

File: src/tasks.ts (hunk 2 above)
Comment (DustinCampbell) [inline src/tasks.ts]: Great catch -- thanks!

**Pull request merged**